src/App.css

```diff
@@ -1,39 +1,173 @@
-.App {
-  text-align: center;
+header {
+  display: flex;
+  align-items: center;
+  gap: 2rem;
+  padding: 0 3%;
+  background-color: #ffff;
+  border-bottom: 1px solid #e8e8e8;
+  margin-bottom: 4%;
+}
+
+.logo {
+  color: #0290ff;
+  font-size: 1.85rem;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.nav {
+  letter-spacing: 1.9px;
+  color: #121212;
+  font-size: 0.813rem;
+  text-decoration: none;
```

Comment:
- [ ] Hey kudos to you, You did a great job on making this styling look awesome but there are still issues you need to address, If you take a look at the original template you will see some differences between your design and the original design 
- [ ] According to the project requirement Your final design should match Zeplin's design (this includes all elements + the same styling Kindly make your design look exactly like the original given design to make your website look professional and user-friendly you can see the screenshot image I attached below.
- [ ] your design 
![Screenshot 2023-08-26 002626](https://github.com/MargaretTechworld/bookstore/assets/92458236/4a109977-fd7b-49df-a9fd-7f9f4005b0b4)

- [ ] original design
![Screenshot 2023-08-26 004520](https://github.com/MargaretTechworld/bookstore/assets/92458236/844419ec-3a39-4076-8c26-84f1ccb71784)

Reply:
**Updated design** 

![image](https://github.com/MargaretTechworld/bookstore/assets/60655304/1a309e08-6230-4fd9-ab02-b2fd62f1e380)